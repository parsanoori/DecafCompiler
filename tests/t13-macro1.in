define SEMICOLON ;
define FOR100 for(i = 0; i < 100; i += 1)

FOR100
Print(i)SEMICOLON
