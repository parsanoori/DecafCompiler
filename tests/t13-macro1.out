for
(
T_ID i
=
T_INTLITERAL 0
;
T_ID i
<
T_INTLITERAL 100
;
T_ID i
+=
T_INTLITERAL 1
)
Print
(
T_ID i
)
;
